Document importing ansible collections in the content guide (#2510)
theforeman/foreman-documentation: guides/common/modules/con_managing-ansible-content.adoc

> [id="Managing_Ansible_Content{context}"]

Reviewer: ```suggestion
[id="Managing_Ansible_Content_{context}"]
```

= Managing Ansible Content

> You can import Ansible collections from several sources to {ProjectServer}.

Reviewer 2: ```suggestion
You can import Ansible collections from multiple sources to {ProjectServer}.
```
Not sure about this. Consider optional

Author: Vale for RH actually flags `multiple` as a word that's too complex.

> 

Reviewer: ```suggestion
```

IMHO a single sentence is not a "real" paragraph. Your call.

Author: I think I'd normally agree with you, but I'm modelling this after other introductions like this one: https://github.com/theforeman/foreman-documentation/blob/9169132a3ec8938066cef709eb135302cd91590c/guides/common/modules/con_managing-container-images.adoc?plain=1#L4

> For information about Ansible integration in {Project}, see {ManagingConfigurationsAnsibleDocURL}[{ManagingConfigurationsAnsibleDocTitle}].

Reviewer 2: ```suggestion
For information about Ansible integration in {Project}, see {ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].
```
My apologies, I missed this before.

> For information about Ansible integration in {Project}, see {ManagingConfigurationsAnsibleDocURL}[Managing Configurations Using Ansible Integration in {Project}].

Reviewer 2: ```suggestion
For information about Ansible integration in {Project}, see {ManagingConfigurationsAnsibleDocURL}[{ManagingConfigurationsAnsibleDocTitle}].
```

> For information about Ansible integration in {Project}, see {ManagingConfigurationsAnsibleDocURL}[Managing Configurations Using Ansible Integration in Red Hat Satellite].

Reviewer: ```suggestion
For information about Ansible integration in {Project}, see {ManagingConfigurationsAnsibleDocURL}[Managing Configurations Using Ansible Integration in {Project}].
```
